.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,18 @@
+name: Pre-commit-license-check
+
+on:
+  pull_request:
+    types:
+      - edited
+      - opened
+      - reopened
+      - synchronize
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
```

Comment:
Can we try to test this without this and just use the system Python?  It might just be enough that way rather than pinning to older Python potentially.